src/java.desktop/share/classes/java/awt/BufferCapabilities.java

```diff
@@ -63,13 +62,18 @@ public BufferCapabilities(ImageCapabilities frontCaps,
     }
 
     /**
-     * @return the image capabilities of the front (displayed) buffer
+     * Returns the imaga capabilities of the front buffer
```

Comment:
Please add a dot after the first sentence, it is used as a summary doc for the method.
Why the text about "displayed" buffer is removed? is it some special "term" like "intermediate" below?

Reply:
Done